docs/docs/how-tos/How to install and configure Keycloak.md

```diff
@@ -0,0 +1,156 @@
+:::
+
+`root`'s password is generated by the `nebari init` command. If you ran this command while following the steps under [Installing Nebari](/getting-started/installing-nebari), you should have seen something like the following in your terminal output:
+
+```
+Securely generated default random password=<alphanumeric-string> for Keycloak root user
+```
+
+The `init` command also saves the root user password to your `nebari-config.yaml` configuration file under the following path:
+
+`security.keycloak.initial_root_password`
+
+After the initial deployment, it is **highly** recommended that you change the Keycloak `root` user password as soon as possible.
+
+
+1. To change the `root` user password, go to your Nebari instance's admin dashboard - e.g., something like `https://{your-nebari-domain}/auth/admin/` and log in with the root password provided.
+
+   ![Nebari admin view - Root Login to Keycloak form](/img/how-tos/keycloak_master_login.png)
+
+2. From there, click on the **Root** dropdown in the top right of the screen, and select **Manage account**.
+
+   ![Keycloak root user page - manage account tab selected](/img/how-tos/keycloak_root_user_manage_account.png)
+
+3. Under **Account Security** click **Signing In**.
+
+   ![Keycloak root user page - account security](/img/how-tos/keycloak_root_user_account_security.png)
+
+4. In the Password section, click the **Update** button. This will guide you through entering your existing root password, and then creating a new password.
+
+   ![Keycloak root user page - account security, update password](/img/how-tos/keycloak_root_user_update_password.png)
+
+:::warning
+The `security.keycloak.initial_root_password` field in `nebari-config.yaml` has no effect after changing the `root` password. If you redeploy Nebari it **will not reset** the password back to the old one (or anything else that might be in the field in your YAML file). We strongly recommend you delete this field to prevent later confusion.
+:::
+
+## Adding a Nebari user
+
+Nebari does not create a Keycloak user for you during deployment, you will need to add a Nebari user manually in order to log in to your Nebari platform. If you have chosen to use GitHub or Auth0 single-sign-on, you must ensure the value you enter in Keycloak under `Username` exactly matches the usernames from GitHub or Auth0, respectively.
+
+### Add user using Keycloak console
+
+To add a Nebari user from the web console for Keycloak, visit https://{your-nebari-domain}/auth/admin/ and log in using the username `root`, as shown above.
+
+Keycloak uses [realms](https://www.keycloak.org/docs/latest/server_admin/#configuring-realms) to separate their permission scopes under the Keycloak admin console. The realm for Nebari is `nebari` and all Nebari users will be part of the `nebari` realm, on the other hand, the realm for the Keycloak admin console is `master`.
```

Comment:
Does it have to be master? I would *highly prefer avoiding non-inclusive terms like master* - could we use something else like: main, captain?
This is a dev-releated broader question so pinging @costrouc